### ISSUE TITLE
FinalPcmAudioFilter forces 2-channel audio output

### ISSUE THREAD

**Issue by ph0t0shop** [sedmelluq/lavaplayer#553]
For some reason, FinalPcmAudioFilter assumes I want 2-channel audio output. There is no way for me to tell it to output single-channel mono audio instead.

**Comment by sedmelluq** [comment]
Yeah this is a current limitation as it was originally made for either local stereo playback or for use with Discord, with no consideration for other use cases. Will need to change this to make the lib usable for more use cases.
